src/index.css

```diff
@@ -56,3 +74,269 @@ code {
 input {
   margin-right: 10px;
 }
+
+:root {
+  --main-acent-color: #0290ff;
+}
+
+.d_flex {
+  display: flex;
+}
+
+.navigation_container {
+  background: #fff;
+  border-bottom: 1px solid #e8e8e8;
+}
+
+.navigation {
+  justify-content: space-between;
+  width: 80%;
+  height: 75px;
+  margin: 0 auto;
+  align-items: center;
+}
+
+.navigation ul {
+  align-items: center;
+  margin: 0;
+  padding: 0;
+}
+
+.books-list-section {
+  margin: 0;
+  padding: 0;
+}
+
+.books-list-section > li > .secondCol {
+  align-items: center;
+  padding: 0;
+}
+
+.books-list-section > li {
+  justify-content: space-between;
+  align-items: center;
+  list-style: none;
+  margin: 10px 0;
+  padding: 1rem;
+  border-radius: 4px;
+  background: #fff;
+  border: 1px solid #e8e8e8;
+}
+
+.navigation > ul > li {
+  list-style: none;
+  padding: 10px 2rem 10px 0;
+}
+
+.books-list-section li > .secondCol > li {
+  list-style: none;
+}
+
+.books-list-section li > .secondCol > li:nth-child(3) {
+  padding-left: 4rem;
+  padding-right: 5rem;
+  line-height: 7px;
+  border-left: 1px solid #e8e8e8;
+}
+
+.percentageVal {
+  margin: 0;
+  font-size: 2rem;
+  color: #121212;
+}
+
+.completed {
+  opacity: 0.5;
+  font-size: 0.875rem;
+  color: #121212;
+}
+
+.books-list-section li > .secondCol > li:nth-child(2) {
+  padding-right: 6rem;
+  padding-left: 0.5rem;
+}
+
+.lastCol {
```

Comment:
Though your design is not bad, this section on your version doesn't look like the one in the design. In order for your design to be perfect please fix the spacing in your button and around it, you should try to match the template as closely as possible which means following the guidelines for spacing around elements.

Please try adding the corresponding CSS property to stick to the design as much as possible. For example, why don't you try adding some padding and/or margin?

Check the image below to have a better understanding.
![Capture2](https://user-images.githubusercontent.com/75709904/146548424-558a18cb-6928-43b6-a64a-17c0a91c5c49.PNG)

Reply:
Added padding